### ISSUE TITLE
a few 'private' things have badly formatting docstrings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3268]
|[<img alt="mwhudson's avatar" src="https://avatars.githubusercontent.com/u/672022?s=50" width="50" height="50">](https://github.com/mwhudson)| @mwhudson reported|
|-|-|
|Trac ID|trac#3268|
|Type|defect|
|Created|2008-06-02 05:21:20Z|
|Branch|https://github.com/twisted/twisted/tree/private-epytext-formatting-3268|

So the API docs got regenerated included docstrings for things named _like _this (see [#3187](https://github.com/twisted/twisted/issues/3187)) and trunk pydoctor soon will too.  A handful of such docstrings in Twisted are not valid epytext, and so when the apidocs buildbot updates its pydoctor it will start failing -- unless you apply the diff that I'm about to attach to this ticket :)

Attachments:

* [private-malformed-twisted-docstrings.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/43e/43ebc8eb20ada4a273c146172a53df03623910b6/947e0954934e583aaf014c1501531d1e5c092efe.diff) (4324 bytes) - added by mwh on 2008-06-02 05:21:45Z - diff fixing a few dodgy bits of epytext

<details><summary>Searchable metadata</summary>

```
trac-id__3268 3268
type__defect defect
reporter__mwh mwh
priority__highest highest
milestone__ 
branch__branches_private_epytext_formatting_3268 branches/private-epytext-formatting-3268
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1212384080000000 1212384080000000
changetime__1213742757000000 1213742757000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [23917]) Branching to 'private-epytext-formatting-3268'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [23918]) Apply private-malformed-twisted-docstrings.diff

refs [#3268](https://github.com/twisted/twisted/issues/3268)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

(In [23919]) Change `APIBuilder` tests to assert that private APIs do appear in the generated docs

refs [#3268](https://github.com/twisted/twisted/issues/3268)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun removed owner<br>@exarkun set status to `new`|
|-|-|

The patch looks good to me.  I made one further change, adjusting an `APIBuilder` test so that it asserts that private docstrings _do_ show up in the output, rather than that they _do **not**._  The [results of building this branch on supported builders](http://buildbot.twistedmatrix.com/boxes-supported?num_builds=1&branch=/branches/private-epytext-formatting-3268) show only unrelated failures.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_9"></a>|@therve set owner to @exarkun|
|-|-|

There is a typo in web2 docstring: intimiately.

You can merge once fixed, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun commented|
|-|-|

(In [23943]) fix a typo

refs [#3268](https://github.com/twisted/twisted/issues/3268)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun set status to `closed`|
|-|-|

(In [23944]) Merge private-epytext-formatting-3268

Author: mwh, exarkun
Reviewer: exarkun, therve
Fixes: [#3268](https://github.com/twisted/twisted/issues/3268)

Update the docstrings of a number of private APIs so that they are
valid epytext (this, necessary since we want pydoctor to include
all APIs in the generated documentation so that private APIs which
are re-exposed as public somewhere else get documented).

Also change one of the release unit tests from asserting that private
APIs are not documented to asserting that private APIs are documented.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
